shared/studio/tabs/auth/state/index.tsx

```diff
@@ -1286,6 +1328,7 @@ export class DraftProviderConfig extends Model({
   webauthnRelyingOrigin: prop<string | null>(null).withSetter(),
 
   requireEmailVerification: prop(true).withSetter(),
+  useCodeVerificationMethod: prop(false).withSetter(),
```

Comment:
Not a blocker, but I kind of like the `Link` and `Code` enum values over a boolean here. Not a blocker because I'm not sure what other kind of method we could have in the future but leaving it open-ended was a conscious choice for me in the schema design. We can go with a boolean for now and update the UI if we ever get a third way though, so definitely not a blocker.